# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../efsd/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- efsd-2004-02-25/efsd/libefsd.c.orig	2004-05-13 18:40:26.982917168 +0200
+++ efsd-2004-02-25/efsd/libefsd.c	2004-05-13 18:42:10.480183192 +0200
@@ -1392,6 +1392,7 @@
       break;
     case EFSD_CMD_CLOSE:
     default:
+      ;
     }
 
   D_RETURN_(NULL);
@@ -1441,6 +1442,7 @@
       break;
     case EFSD_CMD_CLOSE:
     default:
+      ;
     }
 
   D_RETURN_(-1);
@@ -1464,6 +1466,7 @@
       D_RETURN_(ee->efsd_metachange_event.data);
       break;
     default:
+      ;
     }
 
   D_RETURN_(NULL);
--- efsd-2004-02-25/efsd/efsd_main.c.orig	2004-05-13 18:43:42.328220168 +0200
+++ efsd-2004-02-25/efsd/efsd_main.c	2004-05-13 18:46:00.131270920 +0200
@@ -417,7 +417,7 @@
 	{
 	  if (errno == EINTR)
 	    {
-	      D(__FUNCTION__ " select() interrupted\n");
+	      D("%s select() interrupted\n", __FUNCTION__);
 	      FD_ZERO(&fdrset);
 	      FD_SET(famcon.fd, &fdrset);
 	      fdsize = famcon.fd;
@@ -429,7 +429,7 @@
 	    }
 	  else
 	    {
-	      fprintf(stderr, __FUNCTION__ ": select error -- exiting.\n");
+	      fprintf(stderr, "%s: select error -- exiting.\n", __FUNCTION__);
 	      perror("Mhmmm");
 	      exit(-1);
 	    }
@@ -501,6 +501,7 @@
 	    efsd_dca_remove(m->files, famev.filename);
 	  break;
 	default:
+	  ;
 	}
 
       for (cl = efsd_list_head(m->clients); cl; cl = efsd_list_next(cl))
@@ -522,6 +523,7 @@
 		  sort_files = TRUE;
 		  break;
 		default:
+		  ;
 		}
 	    }
 
@@ -777,7 +779,7 @@
 	{
 	  if (errno == EINTR)
 	    {
-	      D(__FUNCTION__ " select() interrupted\n");
+	      D("%s select() interrupted\n", __FUNCTION__);
 
 	      FD_ZERO(&fdrset);
 
--- efsd-2004-02-25/efsd/efsd_filetype.c.orig	2004-05-13 18:42:58.746845544 +0200
+++ efsd-2004-02-25/efsd/efsd_filetype.c	2004-05-13 18:43:10.975986432 +0200
@@ -1333,6 +1333,7 @@
       }
       break;
     default:
+      ;
     }
 
   D_RETURN_(NULL);
--- efsd-2004-02-25/efsd/efsd_monitor.c.orig	2004-05-13 18:46:39.883227704 +0200
+++ efsd-2004-02-25/efsd/efsd_monitor.c	2004-05-13 18:46:51.289493688 +0200
@@ -358,6 +358,7 @@
 	  sort_files = TRUE;
 	  break;
 	default:
+	  ;
 	}
     }
   
--- efsd-2004-02-25/tools/efsdsh.c.orig	2004-05-13 18:47:40.042082168 +0200
+++ efsd-2004-02-25/tools/efsdsh.c	2004-05-13 18:48:24.588310112 +0200
@@ -223,6 +223,7 @@
 			 efsd_metadata_get_str(ee));
 		  break;
 		default:
+		  ;
 		}
 	    }
 	  break;
@@ -353,9 +354,11 @@
 		   efsd_metadata_get_str(ee));
 	    break;
 	  default:
+	    ;
 	  }
       }
     default:
+      ;
     }
   
   /* Cleanup memory allocated for this event */
@@ -373,6 +376,7 @@
       printf("Broken pipe caught.\n");
       break;
     default:
+      ;
     }
 
   exit(-1);
